# Array of grasping points
pr2_grasping/GraspingPoint[] data